desc/plotting.py

```diff
@@ -3753,3 +3760,341 @@ def plot_logo(save_path=None, **kwargs):
         fig.savefig(save_path, facecolor=fig.get_facecolor(), edgecolor="none")
 
     return fig, ax
+
+
+def plot_regcoil_outputs(
```

Comment:
I'm still not super convinced this is needed, since it seems to duplicate a lot of our existing stuff and its trying to do a lot of different things.

IE, for contour plots the user can easily just call plot_2d themselves with the given field/surface returned from regcoil. And for the scatter plots regcoil returns the data needed so I'm not sure we need a function just to do `plt.scatter`

Reply:
that is fair. I will make these helpers in the tutorial and remove this, it is a pain to test anyways